.gitpod.yml

```diff
@@ -0,0 +1,45 @@
+image: gitpod/workspace-full:legacy-dazzle-v1
```

Comment:
This is why things didn't work properly yesterday! Gitpod changed their system for building Docker images, and the latest `gitpod/workspace-full` (used by default) broke some Python stuff. Until it's fixed I've set this to use the old Docker image.